regparser/grammar/terms.py

```diff
@@ -16,6 +16,7 @@
 
 e_tag = (
     Suppress(Regex(r"<E[^>]*>"))
+    + NotAny(Marker("this") + Marker("term")) 
```

Comment:
Gotcha. After some code archeology, I found that this test was added in #154 (the start of the emphasized term pattern). I'm pretty sure the test exists to verify the paragraph-marker's existence -- just to verify that the grammar is working correctly. Since you are taking out that requirement (by making the marker optional), I think you can safely remove the test (and not need the "this term" exclusion)